File: App/JSCode/Protocol/GetModuleBookListRequest.js



/**
 * @author bangwen.lei
 */
import BaseRequest from './BaseRequest'
export default class GetModuleBookListRequest extends BaseRequest {
    //书单ID
    moduleId;//private long 

    //列表起始位置
    start;  // private int

    //列表每页条数
    limit;  // private int 
}

File: JsonData/GetModuleBookListRequest.java
package com.skymobi.payment.nvl.api.model;

import com.skymobi.payment.nvl.api.model.common.BaseRequest;

public class GetModuleBookListRequest extends BaseRequest {

	//书单ID
	private long moduleId;
	
	//列表起始位置
	private int start;
	
	//列表每页条数
	private int limit;



	public long getModuleId() {
		return moduleId;
	}

	public void setModuleId(long moduleId) {
		this.moduleId = moduleId;
	}

	public int getStart() {
		return start;
	}

	public void setStart(int start) {
		this.start = start;
	}

	public int getLimit() {
		return limit;
	}

	public void setLimit(int limit) {
		this.limit = limit;
	}
	
	
}
